include/ticks/mission_prep.hpp

```diff
@@ -17,6 +21,16 @@ class MissionPrepTick : public Tick {
  public:
     explicit MissionPrepTick(std::shared_ptr<MissionState> state);
 
+    std::vector<Bottle> bottlesToDropV;
+    std::array<Bottle, NUMBOTTLES> bottlesToDrop;
+    std::vector<std::pair<cv::Mat, BottleDropIndex>> referenceImages;
+
+    std::string matchingModelPath;
+    std::string segmentationModelPath;
+    std::string saliencyModelPath;
+    
+    std::unique_ptr<Pipeline> pipeline;
```

Comment:
@Tyler-Lentz 
Currently in `mission_prep`  Tick, the pipeline for the CV is locally initialized in the header file. Will there be a scope-problem if that is passed into the CVAggregator? 

Also currently the code fails to compile because the pipeline is a unique ptr.